ExternalLink widget (#1876)
kyma-project/busola: docs/extensibility/display-section.md

<img src="./assets/display-widgets/ControlledBy--kindOnly.png" alt="Example of a ControlledBy widget without name link" width="40%" style="border: 1px solid #D2D5D9">

### ExternalLink

ExternalLink widgets render the link to external page.

#### Widget-specific parameters

> - **linkFormula** - an optional jsonata function to generate custom link.

Reviewer: ```suggestion
- **linkFormula** - an optional JSONata function to generate a custom link.
```
Do we have a link to this JSON function? Maybe we can add it here?

> - **textFormula** - an optional string or item which will be displayed as link.

Reviewer: ```suggestion
- **textFormula** - an optional string or item which is displayed as a link.
```

#### Examples

##### linkFormula and textFormula usage

> ```json

Reviewer: Can you change the code examples to yaml?
See https://github.com/kyma-project/busola/issues/1873 for reference

{
  "source": "$item",
  "name": "spec.servers.port.name",
  "widget": "ExternalLink",
  "linkFormula": "'https://' & $item.port.name & ':' & $string($item.port.number)",
  "textFormula": "$item.port.name"
}
```

<img src="./assets/display-widgets/ExternalLink.png" alt="Example of a ExternalLink widget" width="50%" style="border: 1px solid #D2D5D9">

##### Source only

> ```json

Reviewer: See my previous comment
